packages/react-native-sdk/src/components/Livestream/LivestreamControls/ViewerLivestreamControls.tsx

```diff
@@ -16,13 +16,18 @@ export type ViewerLivestreamControlsProps = ViewerLeaveStreamButtonProps & {
    * Component to customize the leave stream button on the viewer's end live stream.
    */
   ViewerLeaveStreamButton?: React.ComponentType<ViewerLeaveStreamButtonProps> | null;
+  /**
+   * Component to add or customize the right side elements on the viewer's side of live stream.
+   */
+  ViewerLiveStreamControlsRightElement?: React.ComponentType | null;
 };
```

Comment:
lets override the entire ViewerLivestreamControls for consistentcy.. right element seems simple but not consistent with other components like lobby and callcontent

Reply:
Done 👍🏻